Sources/NetworkService/HTTPMethod.swift

```diff
@@ -0,0 +1,11 @@
+/// The HTTP request method.
+public enum HTTPMethod: String {
+    /// The "GET" request.
```

Comment:
nit:

Maybe will be better to describe what that methods do.
For example "Used to read or retrieve a resource" or something like that.

Reply:
I think that is redundant. These are trivial HTTP methods with a lot of documentation, articles and resources which describe these methods better than me.

Reply:
Fixed )